### ISSUE TITLE
Criteria for Feature Selection

### ISSUE THREAD

**Issue by lbotti319** [lbotti319/bayesian_math_scores#6]
How do we want to go about feature selection?

There are a couple things we could look at
* Correlation with target variable
* Scale of CI in MH-Regression
* Anything else somebody comes up with

Additionally we can take an additive approach or subtractive. I don't have a preference
* Only take the top `n` number of features based on above criteria, or features that meet certain thresholds
* Eliminate features that don't meet thresholds based on the above criteria.

What are your thoughts?

**Comment by inespancorbo** [comment]
Updated the notebook in my branch for the correlations. Will PR soon. I opened an issue and suggested failures and higher_yes as those with missing data. I think we can think of which features to select/which to assign missingness in the context of the model itself (both for MAR ignorable and non ignorable -- Aaron has read the most so might have ideas). Other than the 2 variables with missing data, I would just go with 5 other variables? (we can just write in our report that we did this for ease/demonstrability purposes). Otherwise our beta coefficient and convergence diagnostics are going to be too much.
